archinstall/default_profiles/desktops/labwc.py

```diff
@@ -0,0 +1,78 @@
+from typing import TYPE_CHECKING, override
+
+from archinstall.default_profiles.desktops import SeatAccess
+from archinstall.default_profiles.profile import GreeterType, ProfileType, SelectResult
+from archinstall.default_profiles.xorg import XorgProfile
+from archinstall.tui.curses_menu import SelectMenu
+from archinstall.tui.menu_item import MenuItem, MenuItemGroup
+from archinstall.tui.types import Alignment, FrameProperties, ResultType
+
+if TYPE_CHECKING:
+	from collections.abc import Callable
+
+	from archinstall.lib.translationhandler import DeferredTranslation
+
+	_: Callable[[str], DeferredTranslation]
+
+
+class LabwcProfile(XorgProfile):
+	def __init__(self) -> None:
+		super().__init__(
+			'Labwc',
+			ProfileType.WindowMgr,
+			description=''
+		)
+
+		self.custom_settings = {'seat_access': None}
+
+	@property
+	@override
+	def packages(self) -> list[str]:
+		additional = []
+		if seat := self.custom_settings.get('seat_access', None):
+			additional = [seat]
+
+		return [
+			"alacritty",
+			"xorg-xwayland"
```

Comment:
`xorg-wayland` doesn't seem to be a package required for installation https://wiki.archlinux.org/title/Labwc